proseco/guide.py

```diff
@@ -189,20 +185,21 @@ def search_stage(self, stage):
         cand_guis[scol][imp_spoil] += GUIDE_CHAR.errs['hot pix']
         ok = ok & ~imp_spoil
 
-        # Check for star spoilers (by light) background and edge
-        bg_pix_thresh = scoreatpercentile(dark, stage['Spoiler']['BgPixThresh'])
-        reg_frac = stage['Spoiler']['RegionFrac']
-        bg_spoil, reg_spoil = check_spoil_contrib(cand_guis, ok, stars,
-                                                  reg_frac, bg_pix_thresh)
-        cand_guis[scol][bg_spoil] += GUIDE_CHAR.errs['spoiler (bgd)']
-        cand_guis[scol][reg_spoil] += GUIDE_CHAR.errs['spoiler (frac)']
-        ok = ok & ~bg_spoil & ~reg_spoil
-
         # Check for 'direct catalog search' spoilers
         mag_spoil = check_mag_spoilers(cand_guis, ok, stars, n_sigma)
         cand_guis[scol][mag_spoil] += GUIDE_CHAR.errs['spoiler (line)']
         ok = ok & ~mag_spoil
 
+        # Check for star spoilers (by light) background and edge
+        if stage['ASPQ1Lim'] > 0:
```

Comment:
The tests that "image" stars onto an ACAImage and check for photons on the 8x8 or on the background pixels are not super speedy.  Does it make sense to only do those checks in stages where we've opened up the filter to allow nonzero ASPQ1?  Within the check, I've set it to only do the  check on stars that *also* have nonzero ASPQ1.